src/components/CytoscapeGraph/CytoscapeToolbar.tsx

```diff
@@ -64,16 +64,16 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps>
     return (
       <div className={cytoscapeToolbarStyle}>
         <ButtonGroup>
-          <Button onClick={this.zoomIn}>
+          <Button onClick={this.zoomIn} title="zoom in">
```

Comment:
I think we want to use the full-on PF Tooltips. They allow us nicer formatting and font choices (and even icons if we want)

Reply:
I just knew someone would ask for that...